tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -780,27 +776,7 @@
                     <exclude name="testListenerObjectDefinedInDifferentPackage"/>
                 </methods>
             </class>
-            <class name="org.ballerinalang.test.object.ObjectEquivalencyTest">
-                <methods>
-                    <exclude name="testEqOfPublicObjects"/>
-                    <exclude name="testEqOfPublicObjects2"/>
-                    <exclude name="testEqOfPublicObjectsInBalo"/>
-                    <exclude name="testEqOfPublicObjectsInSamePackage"/>
-                    <exclude name="testInherentTypeViolationWithNilType"/>
-                    <exclude name="testNonPublicTypedescEq"/>
-                    <exclude name="testObjectAssignabilityBetweenNonClientAndClientObject"/>
-                    <exclude name="testObjectEqualityWithDefaultConstructor"/>
-                    <exclude name="testObjectEqualityWithRecursiveTypes"/>
-                    <exclude name="testObjectEquivalenceWhenFieldsHaveModuleVisibility"/>
-                    <exclude name="testObjectEquivalenceWhenFieldsHavePublicVisibility"/>
-                    <exclude name="testObjectEquivalencyWithArguments"/>
-                    <exclude name="testObjectMemberOrder"/>
-                    <exclude name="testRuntimeEqPublicObjects"/>
-                    <exclude name="testRuntimeEqPublicObjects1"/>
-                    <exclude name="testSubtypingBetweenNonClientAndClientObject"/>
-                    <exclude name="testTupleMatchWithObjectEquivalency"/>
-                </methods>
-            </class>
+            <class name="org.ballerinalang.test.object.ObjectEquivalencyTest"/>
```

Comment:
I think these are already included in https://github.com/ballerina-platform/ballerina-lang/pull/26797/files#diff-3835b6074874bfd7bcb2e191247640239cbc9c4546822113a142766502ade096R122

Reply:
Removed